src/controllers/auth/index.js

```diff
@@ -0,0 +1,57 @@
+const admin = require("../../lib/admin");
+const firebase = require("../../lib/firebase");
+
+async function createUser(req, res) {
+  const { email, password } = req.body;
```

Comment:
Are we only expecting an email and password for registration at this time?

Reply:
For right now we are just taking in email and password. We had talked about making a multi-step signup that collects additional info. Open for discussing further.